sum.py

```diff
@@ -0,0 +1,6 @@
+sum=0;//variable to store sum
+start = int(input("Enter Start value:"))//inputing start value  
```

Comment:
Provide the comment line above the line about which you will be writing a comment. Something like : 
\# inputing start value  
start = int(input("Enter Start value:"))